README.md

```diff
@@ -1,23 +1,44 @@
 # Best Practices for Discord Bot Lists
-*Bot lists must meet these minimum requirements to be considered a bot list.*
 
 ---
 
-Your bot list:
+*Bot lists must meet these minimum requirements to be considered a bot list.*
+
+## Your bot list must:
 
-1. Must use a custom domain name with a respectable, paid TLD.
-2. Must not be a `[xyz].glitch.me` domain.
+1. use a custom domain name with a respectable, paid TLD.
+2. not be a `[xyz].glitch.me` domain.
    - Glitch is acceptable if:
       - Uptime is immaculate
       - A custom domain name is used (whilst still following point `1`)
-3. Should not display all bots on the homepage.
-4. Must be fully functioning.
+3. be fully functioning.
    - No broken links/buttons.
-5. Must have consistent uptime.
+4. have consistent uptime.
    - Minor downtime for fixing a problem is fine
```

Comment:
Maybe change this to `- Minor downtime for maintenance or fixing issues is fine`